### PR TITLE
add CPython 3.7 to tox envlist and CI configs

### PR DESCRIPTION
**Pull request by mattsb42-aws**
*Issue #, if available:* #151 

*Description of changes:*
CPython 3.0 was released last month. We should start testing it.

I also realized after merging #150 that I forgot to point the pyversions badge at the correct repo..so this fixes that too.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by mattsb42-aws** [comment]
Hold on this...I should have looked at Travis's behavior on my branch first...looks like it can't find 3.7 :/

**Comment by mattsb42-aws** [comment]
Ok; now it's just failing as expected on the pylint checks, fixed in #152

File: .travis.yml [hunk 1]
```diff
@@ -18,6 +18,14 @@ matrix:
...
     - python: 3.6
       env: TOXENV=py36-integ
+    - python: 3.7
+      env: TOXENV=py37-local
+      dist: xenial
+      sudo: true
+    - python: 3.7
+      env: TOXENV=py37-integ
+      dist: xenial
+      sudo: true
```
Comment (johnwalker) [inline .travis.yml]: What is this for?

Comment (mattsb42-aws) [reply .travis.yml]: the short answer is that it's currently necessary to get 3.7 on Travis. https://github.com/travis-ci/travis-ci/issues/9815

**Comment by mattsb42-aws** [comment]
rebased on top of #152

File: .travis.yml (hunk 1 above)
Comment (johnwalker) [reply .travis.yml]: gotcha, thanks!

**Pull request merged**